### PR TITLE
HTML comments: Added integration tests

### LINKED ISSUE #9667: HTML comments: Integration tests with complex features
## Provide a description of the task

Introduce integration tests with complex features such as lists, tables, images, media, blockquote etc.

### PR DESCRIPTION
**Pull request by psmyrek**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Internal (html-support): Added integration tests for `HtmlComment` plugin. Closes #9667.

### CODE REVIEW

**Comment by psmyrek** [comment]
New tests for `Table` integration have been added.

File: packages/ckeditor5-html-support/tests/htmlcomment-integration.js [hunk 1]
```diff
@@ -0,0 +1,1455 @@
...
+import TableEditing from '@ckeditor/ckeditor5-table/src/tableediting';
+import TableCaption from '@ckeditor/ckeditor5-table/src/tablecaption';
+
+import HtmlComment from '../src/htmlcomment';
+
+describe( 'HtmlComment integration', () => {
+	describe( 'integration with BlockQuote', () => {
+		let editor;
+
+		function getEditor( initialData = '' ) {
```
Comment (ma2ciek) [inline packages/ckeditor5-html-support/tests/htmlcomment-integration.js:49]: Can we change all `getEditor` to `createEditor`?

**Review by ma2ciek — APPROVED** [review]
LGTM

File: packages/ckeditor5-html-support/tests/htmlcomment-integration.js (hunk 1 above)
Comment (psmyrek) [reply packages/ckeditor5-html-support/tests/htmlcomment-integration.js]: Done.

**Pull request merged**